kubectl-plugin/README.md

```diff
@@ -22,11 +22,11 @@ You can install the Kuberay kubectl plugin using one of the following methods:
 
 Go to the [releases page](https://github.com/ray-project/kuberay/releases) and download the binary for your platform.
```

Comment:
Maybe we should remove this doc, and move the sections which are useful for users to https://docs.ray.io/en/master/cluster/kubernetes/user-guides/kubectl-plugin.html, and the sections related to dev to  https://github.com/ray-project/kuberay/blob/master/ray-operator/DEVELOPMENT.md

Reply:
SGTM, I'm happy to make another PR for this. Lmk

Reply:
SG. Thank you!

Reply:
https://github.com/ray-project/kuberay/pull/3089